assign_rights/tests.py

```diff
@@ -119,6 +119,21 @@ def test_calculate_dates(self):
         for granted in shell.rightsgranted_set.all():
             self.check_object_dates(granted, request_start_date, request_end_date)
 
+    def test_create_json(self):
+        shell = random.choice(RightsShell.objects.all())
+        request_end_date = random_date().isoformat()
+        request_start_date = random_date().isoformat()
+        start_date, end_date = self.assembler.calculate_dates(shell, request_start_date, request_end_date)
```

Comment:
I'm not sure that this line is strictly necessary - the serializer should handle whatever dates you give it.